server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -26,8 +26,14 @@ def index
       @deployments = Deployment.includes(:organization, :lifecycle_environment, :discovered_host,
                                          :discovered_hosts, :ose_master_hosts, :ose_worker_hosts, :subscriptions,
                                          :introspection_tasks, :foreman_task, :openstack_deployment)
+                                .page(params[:page])
                                 .search_for(params[:search], :order => params[:order]).by_id(params[:id])
-      render :json => @deployments, :each_serializer => Fusor::DeploymentSerializer, :serializer => RootArraySerializer
+      render :json => @deployments,
+             :each_serializer => Fusor::DeploymentSerializer,
+             :serializer => RootArraySerializer,
+             :meta => {:total => Deployment.count,
+                       :total_pages => (Deployment.count / 20.0).ceil
```

Comment:
`total_pages` meta is required by ember -infinity addon. `total` is used to display on `Displaying 20 of 57`